engine/xproviders/xproviders.go

```diff
@@ -0,0 +1,164 @@
+package xproviders
+
+import (
+	"errors"
+
+	"github.com/filecoin-project/storetheindex/api/v0/ingest/schema"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/libp2p/go-libp2p/core/peer"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+// AdBuilder contains fields required for building and signing of a new ad with Extended Providers
+type AdBuilder struct {
```

Comment:
this looks very similar to the `extended_providers` package - do you mean to delete this one?

Reply:
Good spot - artifact after rebase onto the `main`. Cleaned up